parts/motor_test.go

```diff
@@ -0,0 +1,46 @@
+package parts
+
+import "testing"
+
+func TestMotor_SetWheelsRPM(t *testing.T) {
```

Comment:
Pretty nice test.
Perhaps you could have an expected wheel rpm value that you test against.